system-variables.md

```diff
@@ -1,202 +1,909 @@
 ---
 title: 系统变量
-aliases: ['/docs-cn/dev/reference/configuration/tidb-server/mysql-variables/']
+aliases: ['/docs-cn/dev/reference/configuration/tidb-server/mysql-variables/','/docs-cn/dev/tidb-specific-system-variables/','/docs-cn/dev/reference/configuration/tidb-server/tidb-specific-variables/','/tidb/dev/tidb-specific-system-variables/']
```

Comment:
'/tidb/dev/tidb-specific-system-variables/' -> '/zh/tidb/dev/tidb-specific-system-variables' @ran-huang

Reply:
Fixed in https://github.com/pingcap/docs-cn/pull/4045